phoenix-ios/phoenix-ios/views/main/HomeView.swift

```diff
@@ -279,14 +273,21 @@ struct HomeView : MVIView {
 				? Utils.hiddenAmount(currencyPrefs)
 				: Utils.format(currencyPrefs, sat: incomingSat)
 			
+			let unconfirmedBalance = swapInWallet.unconfirmedBalance.sat
+			let weaklyConfirmedBalance = swapInWallet.weaklyConfirmedBalance.sat
+			
 			HStack(alignment: VerticalAlignment.center, spacing: 0) {
 			
-				if swapInRejected != nil {
-					Image(systemName: "zzz")
+				if let _ = swapInWallet.expirationWarningInDays() { // less than 30 days away
```

Comment:
30 days is a bit too much, on Android I use 1 day.

Reply:
Done in 53fe8086300ea4b3f46fb887837a312f06c5e6bd